### PR TITLE
Tw 2088 systemize app bar

### PR DESCRIPTION
**Pull request by KhaledNjim**
## Ticket
- #2088 

## Solution
Added TwakeAppBar component and used in similar app bars

## Test recommendations
**Recommendations for how to test this, or anything else you are worried about?**

## Resolved
- Web:
- 
image
image

image
image

- Android:
- IOS:

### CODE REVIEW

File: lib/pages/chat_details/chat_details_view.dart [hunk 1]
```diff
@@ -36,46 +38,23 @@ class ChatDetailsView extends StatelessWidget {
...
-                      style: Theme.of(context).textTheme.titleLarge,
-                      overflow: TextOverflow.ellipsis,
-                    ),
-                  ),
-                ],
-              ),
+          appBar: TwakeAppBar(
+            backgroundColor: controller.responsive.isMobile(context)
+                ? LinagoraSysColors.material().surface
+                : LinagoraSysColors.material().onPrimary,
```
Comment (hoangdat) [inline lib/pages/chat_details/chat_details_view.dart:44]: why we need to customize it? why not use the default one?

File: lib/widgets/app_bars/twake_app_bar_style.dart [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          ? LinagoraTextStyle.material().bodyLarge1.copyWith(
+                color: Theme.of(context).colorScheme.onSurface,
+                height: 24 / 17,
+              )
+          : Theme.of(context).textTheme.titleLarge?.copyWith(
+                color: Theme.of(context).colorScheme.onSurface,
+                height: 32 / 22,
+              );
+
+  static double toolBarHeight(BuildContext context) =>
```
Comment (hoangdat) [inline lib/widgets/app_bars/twake_app_bar_style.dart:26]: why need to duplicate this function?

File: lib/pages/contacts_tab/contacts_appbar_style.dart
Comment (hoangdat) [inline lib/pages/contacts_tab/contacts_appbar_style.dart:1]: please remove:
`static const Size preferredSizeAppBar = Size.fromHeight(120);`
`static const double toolbarHeight = 56.0;`

File: lib/pages/settings_dashboard/settings/settings_view.dart [hunk 3]
```diff
@@ -29,39 +28,10 @@ class SettingsView extends StatelessWidget {
...
-                        LinagoraSysColors.material().surfaceTint,
-                      ).opacityLayer3,
-                    ),
-                  ),
-                )
-              : null,
-          child: Padding(
-            padding: responsiveUtils.isMobile(context)
-                ? SettingsViewStyle.titlePaddingMobile
-                : SettingsViewStyle.titlePaddingWeb,
```
Comment (hoangdat) [inline lib/pages/settings_dashboard/settings/settings_view.dart:53]: please also remove unnecessary in style too

File: lib/pages/chat_details/chat_details_view.dart (hunk 1 above)
Comment (KhaledNjim) [reply lib/pages/chat_details/chat_details_view.dart:44]: some appbars have different color

image

Comment (hoangdat) [reply lib/pages/chat_details/chat_details_view.dart:44]: please choose `LinagoraSysColors.material().onPrimary;`

Comment (KhaledNjim) [reply lib/pages/chat_details/chat_details_view.dart:44]: for both web and mobile ?

Comment (hoangdat) [reply lib/pages/chat_details/chat_details_view.dart:44]: remove it

**Review by hoangdat — APPROVED** [review]

**Pull request merged**